src/prover_disk.hpp

```diff
@@ -577,7 +571,7 @@ class DiskProver {
     //     Where a < b is defined as:  max(b) > max(a) where a and b are lists of k bit elements
     std::vector<LargeBits> ReorderProof(const std::vector<Bits>& xs_input) const
     {
-        F1Calculator f1(k, id);
+        F1Calculator f1(k, id.data());
```

Comment:
heh, we're hoping that `id` is large enough here :) (it would be nice for the `F1Calculator to take a span in the future)

Reply:
We had the same hope before this PR but i see your point.